test-rs/package.json

```diff
@@ -8,7 +8,8 @@
   "scripts": {
     "wkg": "wkg wit fetch",
     "cargo": "cargo component build",
-    "plug": "wac plug target/wasm32-wasip1/debug/test-rs.wasm --plug ../typescript/ts-typescript.wasm -o target/test.wasm --plug ../typescript-system/target/wasm32-wasip1/debug/typescript_system.wasm",
-    "run": "wasmtime run target/test.wasm a b c"
+    "plug": "wac plug target/wasm32-wasip1/debug/test-rs.wasm -o target/test.wasm --plug ../typescript/ts-typescript.wasm",
+    "plug2": "wac plug target/test.wasm -o target/test2.wasm --plug ../typescript-system/target/wasm32-wasip1/debug/typescript_system.wasm",
```

Comment:
You have to plug one at a time. I had to do this for rust-rust-rust as well. Not specific to JCO.